### PR TITLE
Replay mock client wasn't allowing completes

### PR DESCRIPTION
**Pull request by Sushisource**
Which can happen with partial history

### CODE REVIEW

File: client/src/raw.rs [hunk 1]
```diff
@@ -2,11 +2,15 @@
 //! information we need via interceptors. This module contains the necessary stuff to make that
 //! happen.
 
-use crate::metrics::task_queue_kv;
-use crate::{metrics::namespace_kv, raw::sealed::RawClientLike, LONG_POLL_TIMEOUT};
+use crate::{
```
Comment (bergundy) [inline client/src/raw.rs:5]: Is this relevant to the PR?

**Review by bergundy — APPROVED** [review]

**Review by cretz — APPROVED** [review]
